tensilelite/Tensile/TensileUpdateLibrary.py

```diff
@@ -47,6 +47,7 @@ def UpdateLogic(filename, logicPath, outputPath):
     problemTypeState["DataTypeA"] = problemTypeState["DataTypeA"].value
     problemTypeState["DataTypeB"] = problemTypeState["DataTypeB"].value
     problemTypeState["DataTypeE"] = problemTypeState["DataTypeE"].value
+    problemTypeState["DataTypeAmaxD"] = problemTypeState["DataTypeAmaxD"].value
```

Comment:
Do we need this? AmaxD should follow internal precision type (compute type)

Reply:
Yes you are right, I'm also thinking about this, but I see we still have a DataTypeE so I just follow the code convention. It is OK for me to remove it.